craft.lic

```diff
@@ -128,6 +133,75 @@ class Craft
       smith('trapezoidal wire sieve')
     end
   end
+  
+  def train_enchanting
+    crafting_data = get_data('crafting')
+    rank = DRSkill.getrank('Enchanting')
+
+    if rank <= 25 # Tier 1  Extremely Easy
+      # Buy an abolution sigil and wood totem
+      ensure_copper_on_hand(2000, @settings)
+      order_item(crafting_data['artificing'][@hometown]['stock-room'], 2)
+      order_item(crafting_data['artificing'][@hometown]['stock-room'], 6)
+      stow_hands
+      wait_for_script_to_complete('enchant',[2,'radiant trinket','totem'])
+      dispose_trash('totem')
+    elsif rank <= 50 # Tier 2 - Very Easy
+      # Buy an induction sigil and wood totem
+      ensure_copper_on_hand(2000, @settings)
```

Comment:
💡 There seems to be a pretty clear pattern here (`ensure`, `order`, `order`, `stow`, `wait`, `dispose`). You should consider a helper function.